pyKiwi-test.py

```diff
@@ -3,27 +3,36 @@
 from button import Button
 from size import Size
 from pos import Pos
+from text import Text
 
 import styles
 import orientation
 import colour
 import events
+import pygame
 
 def on_button_click():
     print 'button clicked'
 
 class TestGame(Game):
     def __init__(self):
         super(TestGame, self).__init__()
+        self.view = []
```

Comment:
This file was replaced with pyKiwi-game.py. Name was changed due to the fact that the game class was now in this file. Thus, this is irrelevant and has been removed in a new commit.